lib/glific/communications/message.ex

```diff
@@ -66,7 +66,7 @@ defmodule Glific.Communications.Message do
       sent_at: DateTime.truncate(DateTime.utc_now(), :second)
     })
 
-    Tags.remove_tag_from_all_message(message["contact_id"], ["Not replied", "Unread"])
+    Tags.remove_tag_from_all_message(message["contact_id"], ["not-replied", "unread"])
```

Comment:
should we follow the principle of only alphanumeric for shortcode (no spaces no punctuation). this avoids potential encoding issues

Reply:
so eliminate the "-"